src/ralph/backends/data/fs.py

```diff
@@ -224,12 +208,34 @@ def read(  # noqa: PLR0913
                 }
             )
 
+    def _iter_files_matching_query(
+        self, target: Union[str, None], query: BaseQuery
```

Comment:
perhaps arguments could be explicited in the docstring as "target" doesn't feel self explanatory.

Reply:
Good point) Thanks) Added a mention about the "target" here) I keep it brief to avoid repeating the full explication from the `read` method.